.github/workflows/ci.yml

```diff
@@ -145,6 +145,19 @@ jobs:
               exit 1
             fi
 
+      - name: Check ignored files
+        uses: ./.github/actions/run-docker
+        with:
+          digest: ${{ needs.build.outputs.digest }}
+          version: ${{ needs.build.outputs.version }}
+          run: |
+            # Verify that .dockerignore is working and the
+            # Makefile-os is not in the production container
```

Comment:
what's the downside with `Makefile-os` being in the production container?  (or was that just a useful known file)?

Reply:
It is useless in that container. Everything it does is designed to work on a host outside a container. Excluding it prevents you from accidentally running host commands inside a production container.